tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleDependencies.java

```diff
@@ -261,5 +261,11 @@ public class ExampleDependencies {
     .put("//ldml/units/unitLength[@type=\"*\"]/unit[@type=\"*\"]/unitPattern[@count=\"*\"]", "//ldml/listPatterns/listPattern[@type=\"*\"]/listPatternPart[@type=\"*\"]")
     .put("//ldml/units/unitLength[@type=\"*\"]/unit[@type=\"*\"]/unitPattern[@count=\"*\"]", "//ldml/units/unitLength[@type=\"*\"]/compoundUnit[@type=\"*\"]/compoundUnitPattern")
     .put("//ldml/units/unitLength[@type=\"*\"]/unit[@type=\"*\"]/unitPattern[@count=\"*\"]", "//ldml/units/unitLength[@type=\"*\"]/compoundUnit[@type=\"*\"]/compoundUnitPattern1[@count=\"*\"]")
+    .putAll("//ldml/personNames/sampleName[@item=\"*\"]/nameField[@type=\"*\"]",
+        "//ldml/personNames/personName[@length=\"*\"][@usage=\"*\"][@style=\"*\"][@order=\"*\"]/namePattern",
```

Comment:
Should we also put any .../namePattern[@alt=\"*\"] if we have them? Perhaps a future addition...

Reply:
Ideally the dependency will be auto-detected by TestExampleGenerator.main. Then ExampleDependencies.java will go back to being auto-generated, not hand-edited.